config.h

```diff
@@ -38,6 +38,9 @@ settings_t defaults = {
 .show_indicators = true,
 .word_wrap = false,
 .ellipsize = middle,
+.log_path = "~/dunst.log",
```

Comment:
Obviously, there's a more sensible default path.

Reply:
I made it because me and some other people needed the feature. My friends switched to another notification daemon because dunst doesn't have notification logging actually. Forked it for personal usage, made a pull-request because eh why not maybe you'd find the feature useful (even though the code isn't the best since it's the first time I use C hehe). Maybe at least implement it yourself if you found it useful. Thank you for the code review, I'll see and learn. 

Is just since to have notification logging included in the actual notification daemon.

Reply:
We use notification logs to catch up on messages and notifications we miss. Very handy if you're away for a while and  your computer is still on, or if you need to focus for a while, then you can read the notifications later on(also it's a handy way to see messages people delete, which happens quite a bit in our case). It's very nice to always know that your notifications are saved somewhere.

Reply:
Oops it came all the way up here.